### ISSUE TITLE
Med Post

### ISSUE THREAD

**Issue by philusnarh** [philusnarh/PROJECT#42]
**Sentiment Analysis with Python (Part 1)**
https://towardsdatascience.com/sentiment-analysis-with-python-part-1-5ce197074184

**Comment by philusnarh** [comment]
ECG

https://www.kaggle.com/pharvesh/cnn-for-ptb-database-99-93
https://www.kaggle.com/durgaprasad64/arima-method-for-forecasting-product-sales
https://www.kaggle.com/arthurtok/introduction-to-ensembling-stacking-in-python
https://www.kaggle.com/shayanfazeli/heartbeat/kernels
http://ecgview.org/download.asp
https://www.researchgate.net/post/Where_can_I_find_a_database_for_all_types_of_standard_ECG_and_PCG_signals
https://data.mendeley.com/datasets/7dybx7wyfn/3

**Comment by philusnarh** [comment]
**Overview of feature selection methods**
https://towardsdatascience.com/overview-of-feature-selection-methods-a2d115c7a8f7

**How I built a spreadsheet app with Python to make data science easier**
https://hackernoon.com/introducing-grid-studio-a-spreadsheet-app-with-python-to-make-data-science-easier-tdup38f7

**FC-DenseNet — One Hundred Layers Tiramisu, Fully Convolutional DenseNet **
https://towardsdatascience.com/review-fc-densenet-one-hundred-layer-tiramisu-semantic-segmentation-22ee3be434d5

**A Simple Explanation of the Softmax Function**
https://victorzhou.com/blog/softmax/

**How to Develop an Information Maximizing GAN (InfoGAN) in Keras**
https://machinelearningmastery.com/how-to-develop-an-information-maximizing-generative-adversarial-network-infogan-in-keras/

**Convolutional Neural Networks: A Python Tutorial Using TensorFlow and Keras**
https://www.kdnuggets.com/2019/07/convolutional-neural-networks-python-tutorial-tensorflow-keras.html

**Keras Callbacks Explained In Three Minutes**
https://medium.com/towards-artificial-intelligence/keras-callbacks-explained-in-three-minutes-846a43b44a16

**Sentiment Analysis: a benchmark**
https://towardsdatascience.com/sentiment-analysis-a-benchmark-903279cab44a

**A ConvNet that works on like, 20 samples: Scatter Wavelets**
https://towardsdatascience.com/a-convnet-that-works-on-like-20-samples-scatter-wavelets-b2e858f8a385

**Autoencoders vs PCA: when to use which **
https://towardsdatascience.com/autoencoders-vs-pca-when-to-use-which-73de063f5d7

**Understanding Multi-Label classification model and accuracy metrics**
https://medium.com/towards-artificial-intelligence/understanding-multi-label-classification-model-and-accuracy-metrics-1b2a8e2648ca

**SCRAPER**
**Web Scraping with Python and BeautifulSoup**
https://medium.com/incedge/web-scraping-bf2d814cc572

**Build Simple Web Scraper Using Python & Selenium**
https://medium.com/@joonasvenlinen/build-simple-web-scraper-using-python-selenium-f0376b6cc8fa

**Comment by philusnarh** [comment]
**PCA/ TSNE**
https://www.kaggle.com/sulianova/normalization-logistic-regression-random-forest/notebook

**Unsupervised learning**
https://www.kaggle.com/farhanmd29/unsupervised-learning
https://www.kaggle.com/sashr07/unsupervised-learning-tutorial
https://towardsdatascience.com/a-complete-guide-to-principal-component-analysis-pca-in-machine-learning-664f34fc3e5a
https://www.kaggle.com/samratp/creating-customer-segments-unsupervised-learning

**Computer Vision**
https://towardsdatascience.com/computer-vision-for-beginners-part-1-7cca775f58ef
